packages/styles/src/navigation.scss

```diff
@@ -101,9 +101,29 @@ $navListContainerSubmenu: #{$block}__list-container--submenu;
     }
   }
 
+  @mixin _fd-vertical-nav() {
+    &--vertical {
+      @content;
+    }
+  }
+
+  @mixin _fd-horizontal-nav() {
+    &--horizontal {
+      @content;
+    }
+  }
+
+  @mixin _fd-vertical-nav-with-expander() {
+    &.#{$block}--vertical {
```

Comment:
there's already a SCSS variable for this that you can use:
$navHorizontal: #{$block}--horizontal;